setup/security/authentication/oauth/cognito/index.md

```diff
@@ -0,0 +1,62 @@
+---
+title:  "AWS Cognito"
+sidebar:
+  nav: setup
+---
+
+This page instructs you on how to obtain an OAuth 2.0 client ID and client secret for
+use with your AWS Cognito User Pools.
+
+## Setting up an AWS Cognito App Client
+
+1. Navigate to [https://aws.amazon.com/](https://aws.amazon.com/) and log in with your AWS credentials.
+2. Search for Cognito in the search bar.
+3. Select the user pools you want Spinnaker to use.
+4. At the side bar under "General settings", select "App clients", add a client
+  - Make sure you select "Generate client secret"
+5. After that go to "App integration", then to "App client settings"
+  - Select "Cognito User Pool" as one of the "Enabled Identity Providers"
```

Comment:
This one, and the ones following, should probably be numbered instead of sub-bullets. They're just further steps, right? (The two items _under_ "Check the following" are ok as bullets though, of course.